packages/clerk-js/src/ui/components/UserVerification/AlternativeMethods.tsx

```diff
@@ -0,0 +1,135 @@
+import type { __experimental_SessionVerificationFirstFactor, SignInFactor } from '@clerk/types';
```

Comment:
❓  @panteliselef what do you think about renaming this file to something `UVFactorOneAlternativeMethods` so it can match `UVFactorTwoAlternativeMethods`?

Reply:
I have no problem, there is a great chance that event the UV prefix will be replaced in the coming days as the component name is not final yet